synapse/lib/schemas.py

```diff
@@ -65,6 +65,25 @@
 
 reqValidHttpExtAPIConf = s_config.getJsValidator(_HttpExtAPIConfSchema)
 
+_LayerPushPullSchema = {
+    'type': 'object',
+    'properties': {
+        'url': {'type': 'string'},
+        'time': {'type': 'number'},
+        'iden': {'type': 'string', 'pattern': s_config.re_iden},
+        'user': {'type': 'string', 'pattern': s_config.re_iden},
+        'queue:size': {'type': 'integer', 'default': 10_000,
```

Comment:
Do we want to set a maximum here?

Reply:
Probably don't need to? I'm not sure what a reasonable maximum would be.

Reply:
So the existing chunksize / queue size values can come out to about 3-6gb of memory usage for a given feed if the feeder speed exceeds the writer speed [ based on some real world cortex data ]. This is stored as heap memory which isn't always the quickest ( or guaranteed ) to be reclaimed. I don't know what the utility of allowing an unbound queue size would be, since the memory impacts are not ideal.

The default of 10_000 may be a fine maximum since it prevents a user from getting into a situation that can't be any worse than the existing implementation does.